Dockerfile

```diff
@@ -1,19 +1,10 @@
-FROM gradle:5.5-jdk8 as builder
+FROM node:12.16.3
 
-WORKDIR /app
-ENV GRADLE_OPTS -Dorg.gradle.daemon=false
+WORKDIR /usr/src/app
 
-# Keep gradle dependencies in separate image layer
-COPY build.gradle settings.gradle ./
-RUN gradle classes || echo "ok"
+COPY package.json yarn.lock ./
+RUN yarn
 
-# Copy in the src and build
-COPY src /app/src
-RUN gradle build
+COPY . .
```

Comment:
this is fine since the `.dockerignore` file excludes all files except the ones we want here.  I figured that this would be a good opportunity during the lab to talk about how the `.dockerignore` file works.